### PR TITLE
fix(helloworld-shell): fix hadolint

### PR DESCRIPTION
**Pull request by ace-n**
**N.B:** I have **not** tested this locally.

### CODE REVIEW

**Review by halvards — APPROVED** [review]

**Comment by grayside** [comment]
Could you be more specific about what rule hadolint is flagging? I'm not sure I understand what I need to do to avoid this in the future.

**Comment by ace-n** [comment]
The error message is:
```
./identity-platform/gke/backend/Dockerfile:19 DL3045 warning: `COPY` to a relative destination without `WORKDIR` set.
```

File: helloworld-shell/Dockerfile [hunk 1]
```diff
@@ -44,6 +44,9 @@ RUN set -x && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -
     ca-certificates && \
     rm -rf /var/lib/apt/lists/*
 
+# Create and change to the app directory.
+WORKDIR /app
```
Comment (averikitsch) [inline helloworld-shell/Dockerfile:48]: not needed

File: identity-platform/gke/frontend/Dockerfile [hunk 2]
```diff
@@ -16,8 +16,8 @@
 
 FROM python:3.8
 ENV PORT=8080
+WORKDIR /app
 COPY requirements.txt .
```
Comment (averikitsch) [inline identity-platform/gke/frontend/Dockerfile:20]: Can we just `COPY . ./` here instead of copying twice?

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: helloworld-shell/Dockerfile (hunk 1 above)
Comment (ace-n) [reply helloworld-shell/Dockerfile:48]: Did the linter change? AFAICT this is exactly what `hadolint` was failing on.

File: identity-platform/gke/frontend/Dockerfile (hunk 2 above)
Comment (ace-n) [reply identity-platform/gke/frontend/Dockerfile:20]: Maybe, but I'd argue that's best addressed in a different PR.

(This was supposed to be a "quick" lint fix. 😉 )

Comment (averikitsch) [reply identity-platform/gke/frontend/Dockerfile:20]: Please remove from the PR

File: helloworld-shell/Dockerfile (hunk 1 above)
Comment (averikitsch) [reply helloworld-shell/Dockerfile:48]: I didn't see the second FROM statement. This warning is coming from a new version of hadolint. To address this issue we need to think about pinning to a version. Also please update this to `WORKDIR /`. The workdir /app is confusing with the copy statements below

**Review by averikitsch — CHANGES_REQUESTED** [review]

**Comment by ace-n** [comment]
I made some changes; I don't expect them to fix *all* `hadolint` errors, so I'll address any that pop up.

**Review by averikitsch — APPROVED** [review]

**Pull request merged**